apps/loader/chunked_upload.js

```diff
@@ -79,13 +83,17 @@ async function startUpload(filename) {
 }
 
 function continueUpload(token) {
-  return async function(body) {
-    changeStatus('UPLOAD', 'Uploading chunk at: '+ body.offset +' of size '+ body.chunkSize);
+  return async function(body, file) {
+    let progressValue = Math.floor((body.offset/file.size)*100);
+    $('#upload-progress').css('width', progressValue+'%').attr('aria-valuenow', progressValue).text(progressValue + '%');
+    changeStatus('UPLOAD', 'Uploading chunk at: '+ body.offset/(1024*1024) +'MB of size '+
```

Comment:
```suggestion
    changeStatus('UPLOAD', 'Uploading chunk at: '+ body.offset/(1024*1024) +'MB of total '+
```

This originally referred to the chunk size, which you've correctly identified as useless info to the user. Just want to make it clearer that this new value is the total.